js/luno.js

```diff
@@ -126,6 +126,44 @@ module.exports = class luno extends Exchange {
         return this.parseOrderBook (orderbook, timestamp, 'bids', 'asks', 'price', 'volume');
     }
 
+    async fetchOrder (id, symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        let response = await this.privateGetOrders (this.extend ({
+            'id': id.toString (),
+        }, params));
+        let state = (response.state == 'PENDING') ? 'open' : 'closed';
+        // switch response.state {
+        //     case 'PENDING':
+        //         state = 'open';
+        //         break;
+        //     case 'COMPLETE':
+        //         state = 'closed'; // **OR cancelled**
+        //         break;
+        // }
+        let side = (response.type == 'ASK') ? 'sell' : 'buy';
+        // let Otype = '';
+        // if (response.type = "ASK") {Otype = "sell";} else {Otype = "buy";}
+        return {
+            'id':        response.order_id, // string
+            'datetime':  this.iso8601 (response.creation_timestamp), // ISO8601 datetime with milliseconds
+            'timestamp':  response.creation_timestamp, // Unix timestamp in milliseconds
+            'status':    state,         // 'open', 'closed', 'canceled'
+            'symbol':    'BTC/ZAR',      // symbol
+            'type':      'limit',        // 'market', 'limit'
+            'side':      side,          // 'buy', 'sell'
+            'price':      response.limit_price,    // float price in quote currency
+            'amount':     response.limit_volume,           // ordered amount of base currency
+            // 'filled':     1.0,           // filled amount of base currency
+            // 'remaining':  0.5,           // remaining amount to fill
+            // 'trades':   [ ... ],         // a list of order trades/executions
+            // 'fee':      {                // fee info, if available
+            //     'currency': 'BTC',       // which currency the fee is (usually quote)
+            //     'cost': 0.0009,          // the fee amount in that currency
+            // },
+            'info':     { response },         // the original unparsed order structure as is
```

Comment:
In this line we need to set `'info': response,`... no need to wrap it into a bracketed `{}`-object.